resources/features/VAT.feature

```diff
@@ -0,0 +1,370 @@
+Feature: VAT Calculation
+    The VAT Calculation method will take information from the BI data, VAT Refs and PAYE refs tables calculating various turnovers.
```

Comment:
VAT method should not be ingesting the PAYE refs table, instead it should use the output generated by the PAYE method which in turn should append the values to the enterprise table.